File: addTwoDigits/addTwodigits.py
import doctest

def add_two_digits(n):
    """
    >>> add_two_digits(29)
    11
    >>> add_two_digits(211)
    4
    >>> add_two_digits(1000000)
    1
    """
    return sum([int(digit) for digit in str(n)])


if __name__ == '__main__':
    doctest.testmod()

File: addTwoDigits/addTwoDigits.ts
function addTwoDigits(n: any): number {
    return n.toString().split('').reduce((sum, current) => sum + parseInt(current), 0);
}

console.log(addTwoDigits(29));
console.log(addTwoDigits(111));
console.log(addTwoDigits(1000000));